drivers/syslog/syslog.h

```diff
@@ -294,16 +309,18 @@ int syslog_force(int ch);
  *   for the character driver interface.
  *
  * Input Parameters:
- *   buffer - The buffer containing the data to be output
- *   buflen - The number of bytes in the buffer
+ *   syslog_dev - Handle to syslog device to be used.
+ *   buffer     - The buffer containing the data to be output
+ *   buflen     - The number of bytes in the buffer
```

Comment:
```suggestion
 *   buffer     - The buffer containing the data to be output.
 *   buflen     - The number of bytes in the buffer.
```
nit: Just for consistency.

Reply:
done